### ISSUE TITLE
`react-in-jsx-scope` doesn't work with ES6 imports

### ISSUE THREAD

**Issue by emmenko** [yannickcr/eslint-plugin-react#58]
This doesn't work

```javascript
import React from 'react/addons'

const Button = React.createClass({
  render() {
    return (
      <button {...this.props}>{this.props.children}</button>
    )
  }
})

export default Button
```

but if I define it as a variable it does

```javascript
import r from 'react/addons'
const React = r

const Button = React.createClass({
  render() {
    return (
      <button {...this.props}>{this.props.children}</button>
    )
  }
})

export default Button
```

**Comment by yannickcr** [comment]
Thanks for the failing case. I'll look at this.

**Comment by yannickcr** [comment]
It was related to eslint/eslint#1978 which is fixed in ESLint 0.19.0.

**Comment by emmenko** [comment]
Cool, thanks! :+1:
